portfolio/src/main/webapp/script.js

```diff
@@ -30,6 +30,13 @@ function addRandomGreeting() {
 async function fetchRandomNumber() {
   const responseFromServer = await fetch('/random-number');
   const textFromResponse = await responseFromServer.text();
-  const randomContainer = document.getElementById('random-number-container');
-  randomContainer.innerText = textFromResponse;
+  const numberContainer = document.getElementById('random-number-container');
+  numberContainer.innerText = textFromResponse;
 }
+
+async function fetchRandomPhrase() {
+  const responseFromServer = await fetch('/random-phrase');
+  const jsonData = await responseFromServer.json();
+  const phraseContainer = document.getElementById('random-phrase-container');
+phraseContainer.innerText = jsonData[Math.floor(Math.random() * 5)]
```

Comment:
Using the length of the JSON response is more robust instead of a hardcoded 5. What happens if you change the server response to only return 2 quotes but forget to make a change here? Then a random index could be 4 and give you an index out of bounds error.
```suggestion
phraseContainer.innerText = jsonData[Math.floor(Math.random() * jsonData.length)]
```

Reply:
oh didn't saw that I left that part, thx